NOTICE.txt

```diff
@@ -725,5 +725,35 @@ Legal Terms
 
     http://www.freetype.org
 
-
 --- end of FTL.TXT ---
+
+---
+
+This product contains a modified portion of 'marked', a full-featured markdown parser and compiler, written in JavaScript. Built for speed.
```

Comment:
I don't think we need to say that marked has been modified since we haven't touched the code for it at all. If I've mentioned in the past that I've modified the markdown stuff, that's all been through subclassing the default markdown renderer.